sdk/storage/azure-storage-common/inc/azure/storage/common/version.hpp

```diff
@@ -14,7 +14,7 @@
 #define AZURE_STORAGE_COMMON_VERSION_MAJOR 12
 #define AZURE_STORAGE_COMMON_VERSION_MINOR 0
 #define AZURE_STORAGE_COMMON_VERSION_PATCH 0
-#define AZURE_STORAGE_COMMON_VERSION_PRERELEASE "beta.10"
+#define AZURE_STORAGE_COMMON_VERSION_PRERELEASE ""
```

Comment:
Do we need to undefine this similar to C?
https://github.com/Azure/azure-sdk-for-c/blob/079813f2dcd187da95b173bedef21b3c38974bcb/sdk/inc/azure/core/az_version.h#L33

Here's the previous discussion on the topic:
https://github.com/Azure/azure-sdk-for-c/pull/1336#discussion_r492428630